GW2EIEvtcParser/ParsedData/Skills/SkillItem.cs

```diff
@@ -760,6 +763,11 @@ internal static (long, long) GetArcDPSCustomIDs(int evtcVersion)
             { TakedownSmokescale, "https://wiki.guildwars2.com/images/e/e6/Takedown.png" },
             { PhasePounceWhiteTiger, "https://wiki.guildwars2.com/images/2/20/Phase_Pounce.png" },
             { PhotosynthesizeJacaranda, "https://wiki.guildwars2.com/images/2/2f/Photosynthesize.png" },
+            { EvilEye, "https://wiki.guildwars2.com/images/2/27/Evil_Eye.png" },
+            { TormentingVisionSpinegazer, "https://wiki.guildwars2.com/images/0/0b/Tormenting_Visions.png" },
```

Comment:
Are those skill icons actually missing or is it because you were using an outdated API cache?

Reply:
These don't exist
https://api.guildwars2.com/v2/skills?ids=72603&lang=en
https://api.guildwars2.com/v2/skills?ids=72527&lang=en
The other 3 do, I've added all for completition, should I remove them?
https://api.guildwars2.com/v2/skills?ids=72636&lang=en
https://api.guildwars2.com/v2/skills?ids=72843&lang=en
https://api.guildwars2.com/v2/skills?ids=72851&lang=en

Reply:
If they exist in the API, yes, I'll update the API caches so they should appear naturally.